index.html

```diff
@@ -6,6 +6,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="Self-Taught Software Developers Website">
   <title>Self-Taught Software Developers</title>
+ 
+  <link rel ="icon" type="image/png" href="C:\Users\admin\Documents\GitHub\Self-Taught-Software-Developers.github.io\img\stsd-logo.jpg.png">
```

Comment:
The href is still pointing to your local environment rather than the file within the project.

A solution could look like something "img/stsd-logo.jpg"
"img" to reference the image folder
"stsd-logo.jpg" to reference the file within that folder.

"jpg" and "png" are both image file extension names so you don't need both a suffix to the "stsd-logo" file.

Please let me know if you have any questions I look forward to merging your changes.